source/modules/MEI.cmn.xml

```diff
@@ -2262,6 +2262,53 @@
         model.textPhraseLike.limited.</p>
     </remarks>
   </elementSpec>
+  <elementSpec ident="repeatMark" module="MEI.cmn">
+    <gloss versionDate="2022-05-18" xml:lang="en">repetition mark</gloss>
+    <desc xml:lang="en">
+      An instruction expressed as a combination of text and symbols – segno and coda – typically above, 
+      below, or between staves, but not on the staff.</desc>
+    <classes>
+      <memberOf key="att.common"/>
+      <memberOf key="att.facsimile"/>
+      <memberOf key="att.lang"/>
+      <memberOf key="att.repeatMark.log"/>
+      <memberOf key="att.repeatMark.vis"/>
+      <memberOf key="att.repeatMark.ges"/>
+      <memberOf key="att.repeatMark.anl"/>
+      <memberOf key="model.controlEventLike"/>
```

Comment:
Is `model.controlEventLike` sufficient? As said before some of these symbols may appear on a staff directly.

Reply:
Yes
```xml
<repeatMark func="dalSegno"  place="within">
```

Reply:
I think model.controlEventLike is sufficient.